src/components/OracleEventsTableCondensed.vue

```diff
@@ -7,7 +7,12 @@
       <tbody>
         <tr class="oracle-events-table-condensed__row">
           <th class="oracle-events-table-condensed__header">
-            Query Tx
+            <app-tooltip>
+              Query Tx
+              <template #tooltip>
+                {{ oraclesHints.eventTransactionHash }}
```

Comment:
The previous hints have the exact name as it's label. Could we align to this? Applies for also for naming in other places in this file
```suggestion
                {{ oraclesHints.queryTx}}
```